static/app/utils/number/formatNumberWithDynamicDecimalPoints.tsx

```diff
@@ -4,23 +4,33 @@
  * e.g. 0.0001234 -> 0.00012
  * @param value number to format
  */
-
 export function formatNumberWithDynamicDecimalPoints(
   value: number,
   maxFractionDigits = 2
 ): string {
-  if ([0, Infinity, -Infinity, NaN].includes(value)) {
-    return value.toLocaleString();
+  if (!Number.isFinite(value)) {
+    if (value === Infinity) return '∞';
+    if (value === -Infinity) return '-∞';
+    return 'NaN';
```

Comment:
Based on tests with pt-BR and de-DE, this code would still work with toLocaleString(), but I chose to make everything locale‑agnostic for consistency